homework9_count_numbers_of_letters.py

```diff
@@ -0,0 +1,33 @@
+"""Module providing a function printing python version."""
+
+
+def count_the_number_of_letter(line):
+    """Count the number of letters without last letter."""
+    counter = 1
+    answer = []
+    for i in range(1, len(line)):
+        if line[i] == line[i - 1]:
+            counter += 1
+        else:
+            if counter > 1:
+                answer.append(line[i - 1] + str(counter))
+            else:
+                answer.append(line[i - 1])
+            counter = 1
+    return count_last_letter(line, answer, counter)
```

Comment:
separate function to work with the last character?

Reply:
Yes, it was a creative approach to the solution :)
Changed it.